test/com/facebook/buck/jvm/kotlin/KotlinLibraryIntegrationTest.java

```diff
@@ -63,4 +63,12 @@ public void shouldFailToCompileInvalidKotlinCode() throws Exception {
         workspace.runBuckCommand("build", "//com/example/bad:fail");
     buildResult.assertFailure();
   }
+
+  @Test
+  public void shouldCompileMixedJavaAndKotlinSources() throws Exception {
+    KotlinTestAssumptions.assumeCompilerSupportsMixedSources();
```

Comment:
Can we just use junit's `@Ignore` annotation and link it with a github issue for bumping the compiler? 
Something like `@Ingore("https....github/issues/1234")`